website/documentation/050-tutorials/content/howto/secure-setup/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Hardening the Gardener Community Setup
+description: "Hardening the Gardener Community Setup"
+type: tutorial-page
+level: advanced
+index: 5
+category: Security
+scope: operator
+aliases: ["hardening"]
+---
+
+# Hardening the Gardener Community Setup
+Gardener stakeholders in the Open Source community usually use the [Gardener Setup Scripts](https://github.com/gardener/landscape-setup), to create a Garden cluster based on Kubernetes v1.9 which then can be used to create Shoot clusters based on Kubernetes v1.10, v1.11 and v1.12. Shoot clusters can play the following roles in a Gardener landscape: 
+•	Soil (aka Super Seed) cluster
+•	Seed cluster
+•	Shoot cluster 
+As Alban Crequy from Kinvolk has recommended in his recent Gardener blog [Auditing Kubernetes for Secure Setup](https://gardener.cloud/050-tutorials/content/howto/insecure-configuration/) the Gardener Team at SAP has applied several means to harden the Gardener landscapes at SAP. This blog describes how you can harden your Gardener Community Setup.
+ 
```

Comment:
We apply those setting by default so there should be no need for users to do this. We also don't really have soil clusters here (that would be one level more). if so, the Garden cluster would be the soil cluster.

Reply:
will be changed